chaosaws/__init__.py

```diff
@@ -67,7 +67,29 @@ def aws_client(resource_name: str, configuration: Configuration = None,
         boto3.setup_default_session(
             profile_name=profile_name, region_name=region, **creds)
 
-    return boto3.client(resource_name, region_name=region, **creds)
+    # default config
+    if configuration.get("aws_role_arn") is None:
+        return boto3.client(resource_name, region_name=region, **creds)
+    else:
+        # connect to sts client
+        client = boto3.client('sts', region_name=region, **creds)
+        aws_role_arn = configuration.get("aws_role_arn")
+        # get credentials for the role we want
+        response = client.assume_role(
+            RoleArn=aws_role_arn,
+            RoleSessionName="tempDetourSession")['Credentials']
+        # create new dictionary for credentials
+        newCreds = dict(
```

Comment:
We don't use that style for variable name.

It should be `new_creds` which is the common convention in Python's land ;)

Reply:
will change this :)

Reply:
Thanks! :)